tin/apps/assignments/models.py

```diff
@@ -170,14 +170,14 @@ def save_grader_file(self, grader_text: str) -> None:
         self.grader_file.name = fname
         self.save()
 
-        fpath = os.path.join(settings.MEDIA_ROOT, self.grader_file.name)
+        fpath = settings.MEDIA_ROOT / self.grader_file.name
```

Comment:
Need to test - this is slightly concerning.

Reply:
I'm not sure what is concerning here :thinking:

Reply:
Not the change itself - I'm just a bit sus of how this code block works.